action.yml

```diff
@@ -37,6 +37,6 @@ inputs:
 
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:v0.1.5
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:v0.1.6
```

Comment:
Why would you go from v1 to v0.1.6? Shouldn't it be v1.0.1 at least?

Reply:
If we are considering this as major release then we can tag it as  v1.0.1 .

Reply:
The next major release will be v2.

Reply:
ok will create release as v1.0.1 . Are we still going to keep the tag as v1 .

Reply:
Yes, that is correct.

Reply:
ok can you please approve the PR . I will go ahead and merge .